lib/main.dart

```diff
@@ -4,17 +4,20 @@ import 'package:to_do_list_kamilg/app/app.dart';
 
 void main() async {
   WidgetsFlutterBinding.ensureInitialized();
-  final localStorage = LocalStorage.instance;
+  final localStorage = LocalStorage(databaseName: 'tasks.db');
```

Comment:
databaseName should not be declared in main. It should be declared in tasks_repository.